### ISSUE TITLE
Py3 tests returning ResourceWarning MANY times

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#1414]
This seems to be fine, but we can't live like this:

```
ResourceWarning: unclosed <socket.socket fd=18, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('172.19.0.7', 42974), raddr=('172.19.0.5', 8983)> [six.py:578]
```

That warning appears CONSTANTLY in our tests.

**Comment by flooie** [comment]
oh you noticed

**Comment by mlissner** [comment]
Oh indeed.

**Comment by mlissner** [comment]
I spent some time trying to tackle this yesterday, but failed. It seems like adding this to our settings would do the trick, but nope:

```
warnings.filterwarnings(
    # These should go away as we upgrade more of our stack. Eventually,
    # we'll want to remove this and start figuring out where we need to
    # clean this up.
    "ignore", message="unclosed", category=ResourceWarning
)
```
I'll take another whack at it unless somebody has an idea what's going on here.

**Comment by flooie** [comment]
So - the resource warning error can be filtered into at least two categories, SOLR and SELENIUM.  

It seems that python3, when we connect across docker containers thinks we should close our SOLR connections.  
This was relatively easy to resolve, although difficult to find all the solr connections across tests and the project at large.

**Comment by flooie** [comment]
ResourceWarning has now been eliminated - save two resource warnings at the end of 400+ tests.  

The selenium bug was solved by upgrading to the 4.0.0 alpha.  Urllib.poolmanager was never cleared previously.
